scopes/react/react/react.env.ts

```diff
@@ -219,7 +219,7 @@ export class ReactEnv implements Environment {
    * return a path to a docs template.
    */
   getDocsTemplate() {
-    return require.resolve('./docs');
+    return require.resolve('@teambit/react.ui.docs-app');
```

Comment:
once you changed this to component, you have to make sure the react aspect is dependent on this (otherwise, on runtime with real version you will get `can't find module @eambit/react.ui.docs-app`
add `import @teambit/react.ui.docs-app` at the top or via the component.json